air-quality-ui/e2e/summary.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { expect, test } from "@playwright/test";
```

Comment:
Could we change the name to breadcrumbs.spec.ts or something? I think this is specifically for the breadcrumbs as opposed to the summary page?

Reply:
It shall be done!